xml/telnet_banners.xml

```diff
@@ -816,4 +816,589 @@
+     ksIDAyIE9jdG9iZXIgMjAxOCBsb2dpbjog
+    </example>
+    <param pos="0" name="hw.vendor" value="Turbolinux"/>
+    <param pos="0" name="hw.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^UnixWare ([^ ]+).*$">
+    <description>UnixWare</description>
+    <example _encoding="base64" os.version="2.1.3">
+     VW5peFdhcmUgMi4xLjMgKHByb2ZpbCkgKHB0cy8zKQoKCgpsb2dpbjog
+    </example>
+    <param pos="0" name="hw.vendor" value="SCO"/>
+    <param pos="0" name="hw.family" value="UnixWare"/>
+    <param pos="0" name="hw.device" value="UnixWare"/>
+    <param pos="0" name="hw.product" value="UnixWare"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Telnet Server Build (5.*)">
+    <description>Windows 2000</description>
+    <example _encoding="base64" os.version="5.00.99034.1">
+      TWljcm9zb2Z0IChSKSBXaW5kb3dzIE5UIChUTSkgVmVyc2lvbiA0LjAwIChCdWlsZCAxMzgxKQpXZWxj
+      b21lIHRvIE1pY3Jvc29mdCBUZWxuZXQgU2VydmljZSAKVGVsbmV0IFNlcnZlciBCdWlsZCA1LjAwLjk5MDM0LjEKCmxvZ2luOiA=
+    </example>
+    <param pos="0" name="hw.vendor" value="Microsoft"/>
+    <param pos="0" name="hw.family" value="Windows"/>
+    <param pos="0" name="hw.product" value="Windows 2000"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Welcome. Type return, enter password at # prompt">
+    <description>Brother Printer</description>
+    <example _encoding="base64">
+      V2VsY29tZS4gVHlwZSByZXR1cm4sIGVudGVyIHBhc3N3b3JkIGF0ICMgcHJvbXB0Cg== 
+    </example>
+    <param pos="0" name="hw.vendor" value="Brother"/>
+    <param pos="0" name="hw.family" value="Brother"/>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.product" value="Brother Printer"/>
+  </fingerprint>
+  <fingerprint pattern="^(.*) Copyright by ARESCOM">
+    <description>Arescom System</description>
+    <example _encoding="base64" hw.model="NDS1260HE-TLI">
+      TkRTMTI2MEhFLVRMSSBDb3B5cmlnaHQgYnkgQVJFU0NPTSAyMDAyCgoKClBhc3N3b3JkOgo=
+    </example>
+    <param pos="0" name="hw.vendor" value="Arescom"/>
+    <param pos="0" name="hw.device" value="WAP"/>
+    <param pos="0" name="hw.family" value="UNKNOWN"/>
+    <param pos="1" name="hw.model"/>
+  </fingerprint>
+  <fingerprint pattern="Welcome to ViewStation">
+    <description>Polycom ViewStation Video Vonference System</description>
+    <example _encoding="base64">
+      V2VsY29tZSB0byBWaWV3U3RhdGlvbgoKUGFzc3dvcmQ6
+    </example>
+    <param pos="0" name="hw.vendor" value="Polycom"/>
+    <param pos="0" name="hw.device" value="ViewStation"/>
+    <param pos="0" name="hw.product" value="UNKNOWN"/>
+    <param pos="0" name="hw.family" value="UNKNOWN"/>
```

Comment:
I'm not sure we should be using UNKNOWN like this.  If we don't know a particular field, just leave it out.